vidur/scheduler/replica_scheduler/sarathi_replica_scheduler.py

```diff
@@ -13,53 +13,37 @@ def __init__(self, *args, **kwargs):
         # sarathi config
         self._num_running_batches = 0
         self._preempted_requests = []
-        self._chunk_size = self._config.sarathi_scheduler_chunk_size
-        # club multiple prefills to ensure uniform chunk size
-        self._enable_rolling_prefills = (
-            self._config.sarathi_scheduler_enable_rolling_prefills
-        )
-        # when we are packing multiple prefills in a batch, we need to ensure
-        # that we don't end up packing a very small prefill chunk just to make batch full
-        # because that will lead to reduced number of schedulable prefill requests
-        self._prefill_fitting_tolerance = (
-            self._config.sarathi_scheduler_prefill_fitting_tolerance
-        )
-        # vLLM config
-        self._watermark_blocks_fraction = (
-            self._config.sarathi_scheduler_watermark_blocks_fraction
-        )
         # For vLLM and its derivatives, we only need to set a loose max batch size
         # Memory requirements are handled explicitly by the scheduler
-        self._max_batch_size = self._config.replica_scheduler_batch_size_cap
         self._max_micro_batch_size = (
-            self._config.replica_scheduler_batch_size_cap // self._num_stages
+            self._replica_scheduler_config.batch_size_cap // self._num_stages
```

Comment:
nit: within replica scheduler we can just call this self._config? in cases, where the particular config corresponds to specific class, we can use short names